README.md

```diff
@@ -10,11 +10,11 @@
 [![CDash dashboard](https://img.shields.io/badge/CDash-Access-blue.svg)](http://my.cdash.org/index.php?project=Ginkgo+Project)
```

Comment:
Unrelated, but maybe important: 
```suggestion
[![CDash dashboard](https://img.shields.io/badge/CDash-Access-blue.svg)](https://my.cdash.org/index.php?project=Ginkgo+Project)
```
[Pure HTTP](http://my.cdash.org/index.php?project=Ginkgo+Project) seems to be broken

Reply:
Nice find